ts/Dashboards/Plugins/HighchartsSyncHandlers.ts

```diff
@@ -443,16 +443,45 @@ const configs: {
                     if (chart && chart.series.length) {
                         const cursor = e.cursor;
                         if (cursor.type === 'position') {
-                            const [series] = chart.series.length > 1 && cursor.column ?
-                                chart.series.filter((series): boolean => series.name === cursor.column) :
-                                chart.series;
+                            let [series] = chart.series;
+
+                            // #20133 - Highcharts dashboards don't sync
+                            // tooltips when charts have multiple series
+                            if (chart.series.length > 1 && cursor.column) {
+                                const relatedSeries = chart.series.filter(
+                                    /* eslint-disable max-len */
+                                    (series): boolean => series.name === cursor.column
+                                );
 
+                                if (relatedSeries.length > 0) {
+                                    [series] = relatedSeries;
+                                }
+                            }
 
-                            if (series && series.visible && cursor.row !== void 0) {
-                                const point = series.points[cursor.row - offset];
+                            if (
+                                series &&
+                                series.visible &&
+                                cursor.row !== void 0
+                            ) {
+                                const point = series.points[cursor.row - offset],
+                                    useSharedTooltip = chart.tooltip?.shared;
 
                                 if (point) {
-                                    chart.tooltip && chart.tooltip.refresh(point);
+                                    const hoverPoint = chart.hoverPoint,
+                                        /* eslint-disable max-len */
+                                        hoverSeries = hoverPoint?.series || chart.hoverSeries,
```

Comment:
Shouldn't we follow the accepted rules of linting in this project instead of circumventing it?